### DISCUSSION TITLE
WWIV 5.8 GA Release (5.8.1)

### DISCUSSION THREAD

**Discussion by wwiv** [wwivbbs/wwiv#1614] [General]
We are happy to announce a prerelease (aka release candidate, tech preview) of WWIV 5.8 for Linux, and Windows. 

WWIV 5.8 is the 8th major release after the long-awaited 5.0 release and in addition to native
WWIVnet support (no more DOS dependencies) on all platforms, supports native Fido Technology 
Networking (FTN) Support and fully customizable menus.

### System Requirements:

Win32:  Windows 7 or later using MS Visual C++ 2022 Community Edition.
Linux: GCC 10 (or later) on (Debian11, Centos7 with SCL, or Ubuntu 22.04)
CMake 3.18 or newer

| Title | Link |
| --- | --- |
| Main Web Site | https://www.wwivbbs.org |
| Documentation | http://docs.wwivbbs.org |
| Code | http://github.com/wwivbbs/wwiv |

### Upgrade notes.

Please read upgrade.txt in the distribution for more specific upgrade notes from 5.7

### New Features/Changes since 5.7.1 pre-release.
```
* Lowercase the modulename for fileio
Now that we uppercase the module for wwivbasic so it looks right, we need
to lowercase it for all file I/O. (Fixes #1611)
```

<hr /><em>This discussion was created from the release <a href='https://github.com/wwivbbs/wwiv/releases/tag/'>WWIV 5.8 GA Release (5.8.1)</a>.</em>

**Comment by jimcreitz** [comment]
NOTE!  Windows found a virus in the internal file NETF125.ZIP
